src/actions/resize/IResizeAction.ts

```diff
@@ -1,9 +1,11 @@
 import {IAction} from "../../interfaces/IAction";
 import {GravityParam} from "../../params/gravity/Gravity";
+import {IBackgroundAction} from "../background/IBackgroundAction";
 
 export interface IResizeAction extends IAction{
```

Comment:
The IResizeAction is scheduled for removal in a different PR, can you please undo the changes here?